components/Section.js

```diff
@@ -1,9 +1,9 @@
-const Section = ({ id, children }) => (
-  <section id={id}>
-    <h1>{children}</h1>
+const Section = ({ id, children, style }) => (
+  <section style={style} id={id}>
+    {children}
     <style jsx>{`
       section {
-        width: 100vw;
+        width: auto;
```

Comment:
I think you want the sections without the "big style" to still be `100vw` though right?